registry/config_file_test.go

```diff
@@ -0,0 +1,161 @@
+package registry
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/pkg/homedir"
+)
+
+func pass() {
+	pc := make([]uintptr, 10)
+	runtime.Callers(0, pc)
```

Comment:
Yeah, but it looks inconsistent. If we want this for all tests, we can use `-v`. But why we want it only for one package?